File: app/src/main/java/com/suncity/dailynotices/islib/common/OnItemClickListener.java
package com.suncity.dailynotices.islib.common;


import com.suncity.dailynotices.islib.bean.Image;

/**
 * @author yuyh.
 * @date 2016/8/5.
 */
public interface OnItemClickListener {

    int onCheckedClick(int position, Image image);

    void onImageClick(int position, Image image);
}

File: app/src/main/java/com/suncity/dailynotices/islib/common/OnItemClickListener.kt
package com.suncity.dailynotices.islib.common


import com.suncity.dailynotices.islib.bean.Image

/**
 * @author yuyh.
 * @date 2016/8/5.
 */
interface OnItemClickListener {

    fun onCheckedClick(position: Int, image: Image): Int

    fun onImageClick(position: Int, image: Image)
}
